### ISSUE TITLE
Interpoloate_missing type error

### ISSUE THREAD

**Issue by nfoti** [mne-tools/mne-hcp#50]
I am trying to use `hcp.preprocessing.interpolate_missing` on some epochs data that I have loaded from file. The resulting object is of type `EpochsFIF` which is not in the allowable types for `interpolate_missing` (see [here](https://github.com/mne-tools/mne-hcp/blob/master/hcp/preprocessing.py#L217)).

I was not able to fine a way to convert an `EpochsFIF` object to an `Epochs` object in mne so I'm wondering if the code at the link above is supposed to include `EpochsFIF` somehow (maybe via `EpochsBase`)?

**Comment by jasmainak** [comment]
yeah I think that should be changed to `mne.BaseEpochs` since the `BaseEpochs` class is now public

**Comment by nfoti** [comment]
Thanks. I just submitted a PR with this fix.
